comp/core/flare/flare.go

```diff
@@ -44,6 +47,7 @@ type dependencies struct {
 
 	Log                   log.Component
 	Config                config.Component
+	RCsetting             rcsetting.Component
```

Comment:
Same thing here, these are not remote config settings. Can the `rcsetting` fields in the `dependencies` and `flare` structs be renamed?